prototypes/form-wizard/assets/js/foia.js

```diff
@@ -0,0 +1,5 @@
+var FOIA = {
```

Comment:
If you put this file under `src/foia.js`, it'll will be included in the build pipeline under `assets/js/foia.bundle.js`. That way you can use npm packages. Since it's going through browserify, the modules will also be included in an IIFE closure to avoid polluting the global namespace. Just letting you know that's an option.